### DISCUSSION TITLE
Problem upgrading to 1.19.4/@app 2.4.1

### DISCUSSION THREAD

**Discussion by robert-niestroj** [quasarframework/quasar#13869] [General - Components / Directives / etc]
Hi, i have such a problem when upgrading to 1.19.4/@app 2.4.1. 
I have found on Google that `Rule can only have one resource source` it was suggested to downgrade webpack, but in Quasar that's not an option. Any ideas?
![image](https://user-images.githubusercontent.com/9776384/177260994-a36a2752-14cc-4a75-840f-1bddd164ecab.png)

**Comment by MilosPaunovic** [comment]
Follow upgrade guide guide from the release notes.
[https://github.com/quasarframework/quasar/releases/tag/@quasar/app-v2.4.0](https://github.com/quasarframework/quasar/releases/tag/@quasar/app-v2.4.0)

**Reply by robert-niestroj** [reply]
I did. I even create an empty project to see what it does. The upgrade guide does not mention some changes in quasar.conf.js: 
`const ESLintPlugin = require('eslint-webpack-plugin')`

`chainWebpack (chain) {
        chain.plugin('eslint-webpack-plugin')
          .use(ESLintPlugin, [{ extensions: ['js', 'vue'] }])
      },`

But still no luck.

**Reply by robert-niestroj** [reply]
I have now created an empty Qv1 project, updated all deps and have the same exact error message.
